build_tools/scripts/update_submodules.sh

```diff
@@ -37,8 +37,29 @@ if [ -n "$git_status" ]; then
   exit 1
 fi
 
-echo "Pulling latest from remote"
-git pull
+master_url="$(git remote get-url master)"
+upstream_url="https://github.com/google/iree.git"
+if [ ${master_url} == ${upstream_url} ]
+then
+  echo "Pulling latest from remote"
+  git pull
+else
+  for remote in `git remote -v | awk -F\  '{print $1}' | sort | uniq`
```

Comment:
`sort -u` would also work, and space should be the default separator, so I think you can leave off `-F\ `. I guess someone could have changed `$FS`, so maybe it's still good to specify

```suggestion
  for remote in `git remote -v | awk -F\  '{print $1}' | sort -u`
```

Reply:
Got it. Thanks